src/public/app.js

```diff
@@ -0,0 +1,11 @@
+(function() {
+	let failureCount = 0;
+	const maxFailures = 5;
```

Comment:
this is like a true const for readability. recommend a naming pattern like `MAX_FAILURES`?